addons/account/report/inherited_layouts.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<template id="external_layout_footer" inherit_id="report.external_layout_footer">
+    <xpath expr="//ul[@name='page_numbering']" position="before">
+        <ul t-if="not company.custom_footer" class="list-inline">
+            <li t-if="company.vat">TIN: <span t-field="company.vat"/></li>
```

Comment:
Should the VAT number be printed on reports even when accounting is not installed ?